twolevel_dbm.py

```diff
@@ -515,6 +559,9 @@ def __call__(self, parser, namespace, values, option_string=None):
         sys.exit(f"{args.modelbuild_command} command not found")
 
     inputs = read_csv(args.input)
+    if len(inputs)<2:
+        raise RuntimeError("Provide at least 2 inputs.")
+    #TODO add error if 2level and not 2 rows per input
```

Comment:
This is a good idea, but for clean PRs and git history, can you please break this out to a separate branch/PR